src/tree/azure/GroupingItem.ts

```diff
@@ -47,63 +50,103 @@ export class GroupingItem implements ResourceGroupsItem {
             this.portalUrl = createPortalUrl(resourceGroup.subscription, resourceGroup.id);
         }
 
-        this.subscription = {
+        this.subscription = this.context ? {
             // for v1.5 compatibility
             ...this.context.subscriptionContext,
             ...this.context.subscription,
-        };
+        } : undefined;
+
+        if (this.resourceGroup) {
```

Comment:
Could you explain why this is the way it is?

Reply:
`GroupingItem`s can now appear in both the focus view and the resources view. For the resources view, `id` property will behave the same as before.

However, in the focus view, these group items may not have a subscription and so I need to create an id without the subscription.